pkg/handlers/helper.go

```diff
@@ -5,23 +5,24 @@ import (
 	"fmt"
 	"net/http"
 
-	"github.com/openshift/ocm-agent/pkg/ocm"
 	"github.com/prometheus/alertmanager/template"
 	log "github.com/sirupsen/logrus"
 	"sigs.k8s.io/controller-runtime/pkg/client"
 
+	"github.com/openshift/ocm-agent/pkg/ocm"
+
 	_ "github.com/golang/mock/mockgen/model"
 )
 
 const (
 	AMLabelAlertName           = "alertname"
 	AMLabelTemplateName        = "managed_notification_template"
 	AMLabelManagedNotification = "send_managed_notification"
-	AMLabelAlertSourceName     = "source"
-	AMLabelAlertSourceHCP      = "HCP"
-	AMLabelAlertSourceDP       = "DP"
 	AMLabelAlertMCID           = "_mc_id"
 	AMLabelAlertHCID           = "_id"
+	//AMLabelAlertSourceName     = "source"
+	//AMLabelAlertSourceHCP      = "HCP"
+	//AMLabelAlertSourceDP       = "DP"
```

Comment:
These commented constants can be removed instead?

Reply:
yeah, good to be removed